docs/source/development/linting.md

```diff
@@ -9,34 +9,37 @@ Linting tools check your code for errors such as a missing bracket or line inden
 As a project grows and goes through various stages of development it becomes important to maintain code quality. Using a consistent format and linting your code ensures that it is consistent, readable, and easy to debug and maintain.
 
 ## Set up Python tools
-There are a variety of Python tools available to use with your Kedro projects. This guide shows you how to use [`black`](https://github.com/psf/black) and [`ruff`](https://beta.ruff.rs).
-- **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. `black` can
-check for styling inconsistencies and reformat your files in place.
-[You can read more in the `black` documentation](https://black.readthedocs.io/en/stable/).
-- **`ruff`** is a fast linter that replaces `flake8`, `pylint`, `pyupgrade`, `isort` and [more](https://beta.ruff.rs/docs/rules/).
-  - It helps to make your code compliant to [`pep8`](https://pypi.org/project/pep8/).
-  - It reformats code by sorting imports alphabetically and automatically separating them into sections by
-type. [You can read more in the `isort` documentation](https://pycqa.github.io/isort/).
 
+There are a variety of Python tools available to use with your Kedro projects. This guide shows you how to use [`ruff`](https://beta.ruff.rs).
+
+**`ruff`** is a fast linter and formatter that replaces `flake8`, `pylint`, `pyupgrade`, `isort`, `black` and [more](https://beta.ruff.rs/docs/rules/).
+  - It helps to make your code compliant to [`pep8`](https://pypi.org/project/pep8/).
+  - It reformats code and sorts imports alphabetically and automatically separating them into sections by
+type. You can read more in the [`black` documentation](https://black.readthedocs.io/en/stable/).
```

Comment:
It's weird to link to the Black and isort docs when talking about ruff, does ruff have a page explaining this by any chance? Otherwise I'd even remove these links

Reply:
Seconded. Is this just left over from the original docs in error?